fitbenchmarking/fitbenchmarking
# Change the wordings in Install.py and all related scripts

#### Description of Work

Fixes #127 

#### Testing Instructions

1. Run `python setup.py install externals -s bob` in the terminal/command line to observe the changed message in the log file and the terminal
2. Read the scripts inside fitbenchmarking/build. Any texts containing `services` and `service` have been changed to `packages` and `package`, respectively.
